libs/chain-signatures/contract/src/primitives/participants.rs

```diff
@@ -16,6 +16,7 @@ pub struct ParticipantInfo {
     pub url: String,
     /// The public key used for verifying messages.
     pub sign_pk: PublicKey,
+    // pub ra_quote: RAQuote,
```

Comment:
can this be removed?

Reply:
Yeah, it can be removed for now. We will need it for https://github.com/Near-One/mpc/pull/376 (it's related to [this](https://github.com/Near-One/mpc/pull/376#discussion_r2065690816) comment).